### PR TITLE
Buffs Janitors roundstart

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

lights will be broken roundstart more

## Why It's Good For The Game
gives janitors something to do roundstart

## Testing

yep lights broke

### CODE REVIEW

File: code/modules/power/lights.dm [hunk 1]
```diff
@@ -263,12 +263,12 @@
 	switch(fitting)
 		if("tube")
 			brightness_range = 8
-			if(prob(2))
+			if(prob(50))
```
Comment (GDNgit) [inline code/modules/power/lights.dm:266]: Why not prob(99)?
it's balanced i swear

**Comment by Contrabang** [comment]
This is a small revival of my 'maybe' project, that is a little less intense on the server lol

File: code/modules/power/lights.dm (hunk 1 above)
Comment (Contrabang) [reply code/modules/power/lights.dm:266]: you're right, this is too low. I'll add more randomness to buff it

Comment (GDNgit) [reply code/modules/power/lights.dm]: 👍

**Review by farie82 — APPROVED** [review]

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]
Do rand(50, 60) and ill merge it

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**